src/main/java/software/sigma/bu003/internship/coparts/config/security/JwtService.java

```diff
@@ -0,0 +1,73 @@
+package software.sigma.bu003.internship.coparts.config.security;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import io.jsonwebtoken.io.Decoders;
+import io.jsonwebtoken.security.Keys;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.stereotype.Service;
+
+import java.security.Key;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Function;
+
+@Service
+public class JwtService {
```

Comment:
really it;s not a service rather component or better config let;s discuss

Reply:
let;s discuss